app/templates/signatures/ongoing.hbs

```diff
@@ -1,104 +1,120 @@
-{{page-title (t "to-follow-up")}}
+{{page-title (t "to-follow-up") }}
 
-<KDataTable
-  @scrollable={{true}}
-  @stickyHeader={{true}}
-  @clickable={{false}}
-  @content={{@model}}
-  @page={{this.page}}
-  @size={{this.size}}
-  @numberOfItems={{@model.length}}
-  @totalNumberOfItems={{@model.meta.count}}
-  @onChangeSize={{fn (mut this.size)}}
-  @onChangePage={{fn (mut this.page)}}
->
-  <:header>
-    <th class="auk-u-text-nowrap">{{t "document-name"}}</th>
-    <Utils::ThSortable
-      @label={{t "document-type"}}
-      @field="sign-subcase.sign-marking-activity.piece.document-container.type.label"
-      @currentSorting={{this.sort}}
-      @onChange={{fn (mut this.sort)}}
-    />
-    <th>{{t "minister"}}</th>
-    <Utils::ThSortable
-      @label={{t "date-approved"}}
-      @field="decision-activity.start-date"
-      @currentSorting={{this.sort}}
-      @onChange={{fn (mut this.sort)}}
-    />
-    <th>{{t "status"}}</th>
-    <th></th>
-  </:header>
-  <:body as |signFlow|>
-    {{#let signFlow.signSubcase.signMarkingActivity as |signMarkingActivity|}}
-      {{#let signMarkingActivity.piece as |piece|}}
-        <td class="au-u-flex--spaced-tiny">
-          {{#if piece}}
-            <AuLink @route="document" @model={{piece.id}}>
-              {{piece.name}}
-            </AuLink>
-            {{#if (this.isConfidential piece.accessLevel)}}
-              <AccessLevelPill
-                @accessLevel={{piece.accessLevel}}
-                @isEditable={{false}}
+<AuMainContainer class="l-search-container au-u-background-gray-100" as |m|>
+  <m.sidebar>
+    <div class="au-c-sidebar">
+      <div class="au-c-sidebar__content">
+        <div
+          id="search-subroute-filters-area"
+          class="au-o-box au-o-box--small au-o-flow au-o-flow--tiny"
+        >
+          <div class="au-c-card au-o-box au-o-box--small">
+            <Auk::CheckboxTree
+              @label={{t "status"}}
```

Comment:
This should be "Selecteer Alle" as well and a separate label should be used (as with the minister filter)

Reply:
Would this not be inconsistent with the rest of the code because in Search it doesn't use a seperate label for "Selecter Alle" either?